source/unified-test-format/unified-test-format.rst

```diff
@@ -459,6 +459,14 @@ The structure of this object is as follows:
   is enabled. If false, tests MUST NOT run if authentication is enabled. If this
   field is omitted, there is no authentication requirement.
 
+- ``authMechanism``: Optional string. Specifies an authentication mechanism that
```

Comment:
How are we expected to determine this? Wire version? `saslSupportedMechs`?

Reply:
`db.adminCommand({'getParameter':"1", "authenticationMechanisms": "1"})`.  In PyMongo we actually already retrieved and stored all of the server parameters, so we can just look up `authenticationMechanisms` in that [dict](https://github.com/mongodb/mongo-python-driver/blob/8eb74838db37d85b0051d9258af906b6e9ac44bf/test/__init__.py#L446)